### PR TITLE
Add webhook example

### LINKED ISSUE #176: A webhook example
### Summary

title

### What is the feature request for?

The documentation

### The Problem

theres no webhook example

### The Ideal Solution

Make a example for using webhooks

### The Current Solution

_No response_

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by Suven-p**
## Summary

Fixes #176 
## Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [x] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Review by TAG-Epic — COMMENTED** [review]
Possibly add a docstring to the file to explain things like the other examples

**Review by ooliver1 — APPROVED** [review]
assumed its tested

File: examples/webhook.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import asyncio
+import nextcord
+
+async def send_to_webhook(url, content):
+    # Create a new HTTP session and use it to create webhook object
+    async with aiohttp.ClientSession() as session:
+        webhook = nextcord.Webhook.from_url(url, session=session)
+        await webhook.send(content)
+
+loop = asyncio.get_event_loop()
```
Comment (vcokltfre) [inline examples/webhook.py:14]: This is not the preferred way of doing this in Python anymore. Instead It's recommended you use `asyncio.run()`.

i.e.
```py
asyncio.run(send_to_webhook("url", "Hello, world!"))
```

**Review by vcokltfre — CHANGES_REQUESTED** [review]

File: examples/webhook.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+loop.run_until_complete(send_to_webhook('url', 'Hello, world!'))
```
Comment (ooliver1) [inline examples/webhook.py:15]:
```suggestion
asyncio.run(send_to_webhook("url", "Hello, world!"))
```
cleaned up to a suggestion

**Pull request merged**